src/template-analyzer.ts

```diff
@@ -94,7 +94,11 @@ export class TemplateAnalyzer {
       }
     };
 
-    this._ast = parse5.parseFragment(this.source, opts);
+    if (this.source.includes('<html')) {
```

Comment:
I'm okay with starting out with a basic check but this doesn't account for `<HTML` which is still valid and I know there are some devs write it like that still.

Reply:
I'm not too worried about the commented case. That seems unlikely.

Reply:
ah fair point, so i probably should use a regex after all. was trying to avoid it since this'd be faster, but it won't be if i have to lowercase it all :joy: will update before we merge